assets/src/edit-story/components/canvas/meta/index.js

```diff
@@ -11,21 +11,69 @@ import { useCallback } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { useStory } from '../../../app';
+import { PAGE_WIDTH } from '../../../constants';
+import { useStory, useHistory } from '../../../app';
 import Switch from '../../switch';
-import Delete from './delete.svg';
-import Duplicate from './duplicate.svg';
+import Delete from './delete_icon.svg';
```

Comment:
I think the icons should be removed to their own directory.

Reply:
@barklund ^ this hasn't been actions.

Reply:
As in a subdirectory of this directory or a single central directory? Also, why, if they're not reused?

Reply:
Subdirectory would make sense but it's not too urgent IMO. We can also do that for all the components in some dedicated cleanup/refactor PR.